builtin/providers/azurerm/resource_arm_local_network_gateway.go

```diff
@@ -2,9 +2,9 @@ package azurerm
 
 import (
 	"fmt"
+	"net/http"
 
 	"github.com/Azure/azure-sdk-for-go/arm/network"
-	"github.com/Azure/azure-sdk-for-go/core/http"
```

Comment:
I noticed this is still in some of the provider code, but no longer exists in the main repo. We should probably remove this throughout, and get rid of core/tls in the vendor dir too. (doesn't need to be in this PR though)